torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -321,8 +321,7 @@ void pushProfilingCallbacks() {
             try {
               cudaStubs()->record(nullptr, &ctx_ptr->cuda_event_start_, nullptr);
             } catch (const std::exception& e) {
-              C10_LOG_EVERY_N(WARNING, 1000) << "Failed to record CUDA event. "
-                                            << e.what();
+              LOG(WARNING) << "Failed to record CUDA event. " << e.what();
```

Comment:
So this change logs every time unlike pervious? I assume this is ok because logging happens only in case of exceptions?

Reply:
yes, unfortunately I had to avoid using LOG_EVERY_N because of the dependency on some old glog version which doesn't work with the newer clang in some of the internal builds